tests/search/tensor_eval/unstable.sd

```diff
@@ -40,6 +40,10 @@ search multi {
       }
       rank-score-drop-limit: 1.0
     }
+    function output_tensor() {
+      expression: attribute(mybytes)
+    }
+    summary-features: output_tensor
```

Comment:
why not just 'attribute(mybytes)'? I am a bit unsure if this will even work without using 'rankingExpression(output_tensor)' as feature...